http_uniresolver/__init__.py

```diff
@@ -2,12 +2,34 @@
 
 from aries_cloudagent.config.injection_context import InjectionContext
 from aries_cloudagent.resolver.did_resolver_registry import DIDResolverRegistry
-
+import yaml
+import os
 from .http_universal import HTTPUniversalDIDResolver
 
+CONFIG_FILE = "http_uniresolver/default_config.yml"
+
 
 async def setup(context: InjectionContext):
     """Setup the plugin."""
+    # Load config
+
+    conf = context.settings.get("plugin_config", {}).get("http_uniresolver")
+    if conf:
+        with open(CONFIG_FILE, "r") as default_config:
+            config = yaml.safe_load(default_config)
+
+        endpoint = conf.get("endpoint")
+        if endpoint:
+            config["endpoint"] = endpoint
+
+        methods = conf.get("methods")
+        if methods:
+            config["methods"] = methods
+
+        with open(CONFIG_FILE, "w") as default_config:
+            default_config.write(yaml.safe_dump(config))
+
+    # Inject plugin
```

Comment:
Why aren't we just using the plugin configuration values directly? Loading any yaml file should no longer be required in the plugins. In fact, we should remove pyyaml as a dependency. We should implement sane defaults if no configuration is given or else error out and say that configuration is required.

Reply:
What do you suggest to send the configuration to the plugin?  json due internal python json lib? .env file? environment variables?